prediction_market_agent/agents/microchain_agent/nft_treasury_game/deploy_nft_treasury_game.py

```diff
@@ -106,6 +109,7 @@ def load(self) -> None:
             )
 
         super().load()
+        self.import_actions_from_memory_from = get_start_time_of_current_round()
```

Comment:
If the pod restarts, import memory only from the beginning of the current round so as to not confuse the agent.